src/main/java/edu/wpi/grip/core/operations/composite/FilterContoursOperation.java

```diff
@@ -0,0 +1,110 @@
+package edu.wpi.grip.core.operations.composite;
+
+import com.google.common.eventbus.EventBus;
+import edu.wpi.grip.core.InputSocket;
+import edu.wpi.grip.core.Operation;
+import edu.wpi.grip.core.OutputSocket;
+import edu.wpi.grip.core.SocketHint;
+
+import java.io.InputStream;
+import java.util.Optional;
+
+import static edu.wpi.grip.core.operations.composite.FindContourOperation.Contours;
```

Comment:
It's used by two operations and has its own preview view. Other people may want to use it in their python scripts. It should be its own class.